pkg/apis/extensions/v1beta1/conversion.go

```diff
@@ -502,3 +504,45 @@ func Convert_networking_NetworkPolicyList_To_v1beta1_NetworkPolicyList(in *netwo
 	}
 	return nil
 }
+
+func Convert_v1beta1_IngressSpec_To_networking_IngressSpec(in *extensionsv1beta1.IngressSpec, out *networking.IngressSpec, s conversion.Scope) error {
+	if in.Backend != nil {
+		if err := s.Convert(&in.Backend, &out.DefaultBackend, 0); err != nil {
```

Comment:
this can silently rely on reflective conversion, which is very slow. use `Convert_v1beta1_IngressBackend_To_networking_IngressBackend` instead (similar comment applies to all other uses of s.Convert in the new functions)